src/components/Select/Select_shim.css

```
@@ -191,6 +193,14 @@ span.neo-multiselect-combo__header .neo-multiselect__padded-container {
 	cursor: not-allowed !important;
 }
 
+.neo-multiselect
+	.neo-multiselect-combo__header
+	.neo-multiselect__padded-container.highlighted:not(
+		:has(.neo-multiselect-combo__buttons-container)
+	) {
+	background-color: #accef7;
```

Not sure what this color is in our system, but you _must_ use our own variables when adding color. 

I'm not seeing `accef7` in our system, but our color variables look like this:
```suggestion
	background-color: var(--neo-color-blue-500);
```

---

Will ask Design on this.